files/es/learn/html/introduction_to_html/the_head_metadata_in_html/index.md

```diff
@@ -87,13 +87,13 @@ Ya hemos visto el elemento {{htmlelement("title")}} en acción — se puede usar
 
 2. Ahora abre el archivo en tu navegador. Deberías ver algo como esto:
 
-   ![Una sencilla página web con el título configurado a <title> element, y el <h1> configurado a <h1> element.](https://mdn.mozillademos.org/files/12323/title-example.png)Ahora debería quedar claro dónde aparece el contenido de `<h1>` y dónde aparece el contenido de `<title>`.
+   ![Una sencilla página web con el título configurado a \<title> element, y el \<h1> configurado a \<h1> element.](https://mdn.mozillademos.org/files/12323/title-example.png)Ahora debería quedar claro dónde aparece el contenido de `<h1>` y dónde aparece el contenido de `<title>`.
```

Comment:
The image is not rendered. 

```suggestion
   ![Una sencilla página web con el título configurado a \<title> element, y el \<h1> configurado a \<h1> element.](title-example.png)
   Ahora debería quedar claro dónde aparece el contenido de `<h1>` y dónde aparece el contenido de `<title>`.
```

Reply:
Commit this suggestion is required.

Reply:
I believe those files were brought in for the en-US content, but that file isn't local in this repo https://github.com/mdn/translated-content/tree/main/files/es/learn/html/introduction_to_html/the_head_metadata_in_html

Reply:
yes, it is inherited from the `content` repository, yari takes care of that.

Reply:
but let's fix it on other pr, thanks for contributing!

Reply:
OK, i've added those changes, but it's probably better to do a big cleanup once the Markdown conversion is done